navigation/src/androidMain/kotlin/com/freeletics/khonshu/navigation/NavigationSetup.kt

```diff
@@ -55,8 +61,13 @@ public fun NavigationSetup(navigator: ActivityResultNavigator) {
         }
     }
 
-    LaunchedEffect(lifecycleOwner, hostNavigator, navigator) {
-        navigator.collectAndHandleNavEvents(lifecycleOwner.lifecycle, hostNavigator, activityLaunchers)
+    LaunchedEffect(lifecycleOwner, hostNavigator, context, navigator) {
```

Comment:
It was meant to be `activityStarter` instead of `context` which is actually used within the effect. (I had context in there at first). [87ba435](https://github.com/freeletics/khonshu/pull/826/commits/87ba43572cda56058240256488bad07c308f632e)